common/app/model/Badges.scala

```diff
@@ -20,11 +50,13 @@ object Badges {
 
   val rio2016 = Badge("sport/rio-2016", Static("images/badges/rio-2016.svg"))
 
-  val allBadges = Seq(usElection, ausElection, voicesOfAmerica, rio2016, euElection, euRealityCheck, euBriefing, euSparrow)
+  val special = SpecialBadge("4dae5700e6b6fdf66d1567769b41c1c2")
+
+  val allBadges = Seq(usElection, ausElection, voicesOfAmerica, special, rio2016, euElection, euRealityCheck, euBriefing, euSparrow)
 
-  def badgeFor(c: ContentType) = c.tags.tags.map(_.id).foldLeft(None: Option[Badge]) { (maybeBadge, tag) =>
-      maybeBadge orElse allBadges.find(b => b.seriesTag == tag)
+  def badgeFor(c: ContentType) = c.tags.tags.map(_.id).foldLeft(None: Option[(BaseBadge, String)]) { (maybeBadge, tag) =>
```

Comment:
we can change this tomorrow to return a `Badge` instead of a `(BaseBadge,String)` by turning SpecialBadge into a Badge, thanks to @TBonnin for the suggestion